File: twiml/voice/transcription/transcription-inboundtracklabel-outboundtracklabel/transcription-inboundtracklabel-outboundtracklabel.py
from twilio.twiml.voice_response import VoiceResponse, Start, Transcription

response = VoiceResponse()
start = Start()
start.transcription(
    status_callback_url='https://example.com/your-callback-url',
    inbound_track_label='agent',
    outbound_track_label='customer')
response.append(start)

print(response)

File: twiml/voice/transcription/transcription-inboundtracklabel-outboundtracklabel/transcription-inboundtracklabel-outboundtracklabel.js
const VoiceResponse = require('twilio').twiml.VoiceResponse;

const response = new VoiceResponse();
const start = response.start();
start.transcription({statusCallbackUrl: 'https://example.com/your-callback-url', inboundTrackLabel: 'agent', outboundTrackLabel: 'customer'});

console.log(response.toString());
